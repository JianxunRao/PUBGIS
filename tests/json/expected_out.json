{"name": "username", "game": null, "team": "best_team", "positions": [[0, [3079, 4034]], [1, [3095, 4061]], [2, null], [3, [3123, 4108]], [4, [3140, 4138]], [5, [3160, 4170]], [6, [3178, 4201]], [7, [3194, 4228]], [8, [3208, 4251]], [9, [3228, 4285]], [10, [3247, 4318]], [11, [3265, 4348]], [12, [3281, 4374]], [13, [3298, 4402]], [14, [3314, 4429]], [15, [3330, 4459]], [16, [3350, 4491]], [17, [3371, 4525]], [18, [3387, 4551]], [19, [3400, 4574]], [20, [3413, 4598]]]}