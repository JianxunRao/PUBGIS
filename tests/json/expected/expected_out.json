{"name": "username", "positions": [[3079, 4034], [3095, 4061], [3108, 4082], [3123, 4108], [3140, 4138], [3160, 4170], [3178, 4201], [3194, 4228], [3208, 4251], [3228, 4285], [3247, 4318], [3265, 4348], [3281, 4374], [3298, 4402], [3314, 4429], [3330, 4459], [3350, 4491], [3371, 4525], [3387, 4551], [3400, 4574], [3413, 4598]]}